contracts/test-utils/src/types.ts

```diff
@@ -201,3 +201,14 @@ export enum ExchangeFunctionName {
     SetProtocolFeeCollectorAddress = 'setProtocolFeeCollectorAddress',
     DetachProtocolFeeCollector = 'detachProtocolFeeCollector',
 }
+
+export enum ValidatorWalletAction {
```

Comment:
Do we use these anywhere other than the signature validator tests?

Reply:
nope, good catch
will move